Book-A-Desk.Api.Tests/BookingsTests.fs

```diff
@@ -32,25 +35,27 @@ let offices =
 let mockGetOffices () =
     offices
     
-let domainName = "domain.com"
+let domainName = "broadsign.com"
```

Comment:
We should avoid hardcoding the domain as it is an open source project, that's why we put domain.com previously

Reply:
Fixed.